### PR TITLE
Cleaner and more readable code SavingAndLoadingProfileCloud C# example

### PR DESCRIPTION
**Pull request by codingyourlife**
Improved some of the criticised code parts months ago here: https://github.com/Emotiv/community-sdk/issues/73
Checked if all still compiles, changes should have no effect on functionality.

But all together the example STILL makes no sense. I mean what is this userCloudID and where do I get it? What is even being saved/loaded and where is it stored (EC_LoadUserProfile now returns an integer for enum instead of boolean, I expect it to return a profile or what have you)
BTW if version is always -1 why not make it a default value for the parameter?
Glad I don't really need this example for my project...

### CODE REVIEW

File: examples/C#/SavingAndLoadingProfileCloud/Program.cs [hunk 1]
```diff
@@ -1,28 +1,25 @@
-﻿/****************************************************************************
+/****************************************************************************
 **
 ** Copyright 2015 by Emotiv. All rights reserved
 ** Example - SavingAndLoadingProfileCloud
-** How to saving and loading prfile from Emotiv Cloud
+** How to save and load profile from Emotiv Cloud
```
Comment (codingyourlife) [inline examples/C#/SavingAndLoadingProfileCloud/Program.cs]: Could anyone write a bit more what exactly this example does? What information is needed to store/load and what information IS stored/loaded. What can it be used for?

File: examples/C#/SavingAndLoadingProfileCloud/Program.cs [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
         static int engineUserID = -1;
-        static int userCloudID  = 0;
-        static string userName  = "Your account name";
-        static string password  = "Your password";
+        static int userCloudID = 0;
+        static string userName = "Your account name";
+        static string password = "Your password";
         static string profileName = "EmotivProfile";
-        static int version	= -1; // Lastest version
+        static int version = -1; // latest version
```
Comment (codingyourlife) [inline examples/C#/SavingAndLoadingProfileCloud/Program.cs]: This should be a default value for that parameter

File: examples/C#/SavingAndLoadingProfileCloud/Program.cs [hunk 3]
```diff
@@ -1,28 +1,25 @@
...
 
 namespace SavingAndLoadingProfileCloud
 {
     class SavingAndLoadingProfileCloud
     {
         static int engineUserID = -1;
-        static int userCloudID  = 0;
-        static string userName  = "Your account name";
-        static string password  = "Your password";
+        static int userCloudID = 0;
```
Comment (codingyourlife) [inline examples/C#/SavingAndLoadingProfileCloud/Program.cs]: What is a userCloudID and where can I find it!?

**Comment by hoangphamemotiv** [comment]
Hi @codingyourlife 
Thank you so much for your feedback. i missed #73 

- i agree that int version = -1 should be a default parameter. 
- cloud userID is number value of your Emotiv ID, we manage Cloud User follow that number, not EmotivID (your account name). 

- EC_GetAllProfileName(cloudUserID);
// Loading information of all profiles from Cloud to buffer and return number of profile in the list. 
i agree that name of this function is a bad name. but, we are using this function in many our project, so, we havent changed it yet. we will plan to fix that name in next version. 

Thanks.

**Comment by codingyourlife** [comment]
Okay for me it's clear now. You should add a comment explaining that variable for others. I commented it on your master commit [here](https://github.com/Emotiv/community-sdk/commit/adde3a2adf935ebf011beebf81bc6891060737c0) so you know what I mean. I'll close this pull request because your changes now seem to do pretty much the same

**Pull request closed without merging**